forms/Makefile

```
@@ -0,0 +1,60 @@
+	cp ${API_DIR}/.htaccess ${DIST_DIR}/api/; \
+	\
+	# Copy built SPA + .htaccess
+	cp -r ${WEB_DIST_DIR}/* ${DIST_DIR}/web/; \
+	cp ${WEB_DIR}/.htaccess ${DIST_DIR}/web/; \
+	\
+	# Root .htaccess and .env
+	cp .htaccess ${DIST_DIR}/; \
+	cp ${ENV_FILE} ${DIST_DIR}/; \
+	echo "Build completed successfully"
+
+.PHONY: deploy
```

checkmake (0.2.2)

[warning] Missing required phony target "clean"

(minphony)